src/riak_kv_pb_timeseries.erl

```diff
@@ -64,6 +64,7 @@
 -define(E_MISSING_TS_MODULE, 8).
 -define(E_DELETE,   9).
 -define(E_GET,     10).
+-define(E_BAD_KEY_LENGTH, 11).
```

Comment:
I'm all for it (that's what I'm kind of suggesting in the comment above these defines). However, working within the scope of this PR, let's leave it as is for now. Whoever undertakes range assignment, will do so globally and in a separate PR.